justfile

```
@@ -59,7 +59,13 @@ install-denv:
 
 # configure how ldmx-sw will be built
 configure *CONFIG:
-    denv cmake -B build -S . {{ CONFIG }}
+    denv cmake -B build -S .  -DADDITIONAL_WARNINGS=ON  -DENABLE_CLANG_TIDY=ON  {{ CONFIG }}
+
+configure-force-error *CONFIG:
+    just  configure -DWARNINGS_AS_ERRORS=ON {{ CONFIG }}
+
+configure-clang-lto:
+    denv cmake -B build -S . -DADDITIONAL_WARNINGS=ON  -DENABLE_CLANG_TIDY=ON  -DENABLE_LTO=ON  -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang 
```

Is falling back to `denv` necessary here?

```suggestion
configure-clang-lto: (configure -DENABLE_LTO=ON -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang) 
```

---

Done as 
```
configure-clang-lto: (configure "-DENABLE_LTO=ON -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang")
```